backend/data/create_africa_geojson.py

```diff
@@ -0,0 +1,104 @@
+"""
+
+This is a data preprocessing script that is not part of the main system.
+
+We take the raw GeoJSON data from natural earth vectors and extract
+the countries we want + the disputed territories that we want on the map.
+
+This script requires the following two data sources in the same directory as it.
+These are not committed to the repo because they're too big!
+    https://github.com/nvkelso/natural-earth-vector/blob/master/
+        geojson/ne_50m_admin_0_countries.geojson
+
+    https://github.com/nvkelso/natural-earth-vector/blob/master/
+        geojson/ne_50m_admin_0_breakaway_disputed_areas.geojson
+
+"""
+
+import csv
+import json
+
+
+def load_iso_name_dict():
+    """
+    Read the CSV file from our v-dem based dataset and return
+    a dict mapping ISO A3 codes to country names
+    """
+    csv_name_index = 0
+    csv_iso_code_index = 1
+    iso_name_dict = {}
+    filename = 'lieberman_afr_data.csv'
+    with open(filename, encoding='utf-8') as democracy_score_file:
+        reader = csv.reader(democracy_score_file, delimiter=',')
+        _ = next(reader)  # skip headers
```

Comment:
I don't think that you need to bind `next(reader)` to anything.

Reply:
Yep, that's right -- that said, my habit in these situations is to use the underscore to signal that I'm intentionally ignoring the return value of this statement.